### PR TITLE
Updating README.

### PR DESCRIPTION
**Pull request by neverendingqs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
+  * The endpoint that contains the resource of interest (e.g. `https://myschool.brightspace.com/d2l/api/lp/1.0/users/whoami`)
+
+### Running the sample
+
+```
+npm install
+npm start
+```
+
+## Contributing
```
Comment (AlexBedley) [inline README.md]: Forgot to update title?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
 
-	{
-	    "CLIENT_ID": "4b00c5f8-9331-45ff-9664-dcdd36dc4522",
-	    "CLIENT_SECRET": "thesecret",
-	    "AUTH_SITE": "https://auth.brightspace.com",
-	    "TOKEN_PATH": "/core/connect/token",
-	    "AUTHORIZATION_PATH": "/oauth2/auth",
-	    "RESOURCE_ENDPOINT": "<resource_endpoint>"
-	}
+## How to run this sample
```
Comment (AlexBedley) [inline README.md]: Do we want to mention you must register the application with the correct callback url?

File: README.md [hunk 3]
```diff
@@ -1,17 +1,41 @@
...
+* CLIENT_SECRET
+  * The OAuth 2.0 client secret (available from the Brightspace OAuth 2.0 registration page)
+* RESOURCE_ENDPOINT
+  * The endpoint that contains the resource of interest (e.g. `https://myschool.brightspace.com/d2l/api/lp/1.0/users/whoami`)
+
+### Running the sample
+
+```
+npm install
+npm start
```
Comment (AlexBedley) [inline README.md]: Should we also add "navigate to localhost:3000"?

File: README.md (hunk 1 above)
Comment (neverendingqs) [inline README.md]: I meant for it to be `##` (if that's what you meant) (previous heading is `## How to run this sample`).

**Comment by AlexBedley** [comment]
lgtm

**Pull request merged**